src/Halogen/Data/Slot.purs

```diff
@@ -112,13 +117,14 @@ slots
   => Proxy sym
   -> SlotStorage slots slot
   -> Map s (slot query output)
-slots sym (SlotStorage m) = Map.foldSubmap Nothing Nothing go m
+slots sym (SlotStorage m) = mapMaybeWithKeyToNewKey go m
   where
   key = reflectSymbol sym
 
+  go :: Tuple String (OrdBox Any) -> Any -> Maybe (Tuple s (slot query output))
   go (Tuple key' ob) val
-    | key == key' = Map.singleton (unOrdBox (coerceBox ob)) (coerceVal val)
-    | otherwise = mempty
```

Comment:
If this is what's giving you the error, you can just do `Map.empty` here instead of `mempty` but otherwise leave things as the were.